compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -649,6 +649,12 @@ error.annotation.invalid.type=\
   annotation declaration requires a subtype of ''true'', ''map<value:Cloneable>'' or ''map<value:Cloneable>[]'', but \
   found ''{0}''
 
+error.array.size.greater.that.2147483637.not.yet.supported=\
+  array size greater that ''2147483637'' not yet supported
+
+error.invalid.array.length=\
+  invalid array length
```

Shall we say something like

```suggestion
  invalid array length: array length should be a positive integer
```